README.md

```diff
@@ -1,81 +1,71 @@
-
 Optimized Einsum: A tensor contraction order optimizer
 ======================================================
 
-Optimized einsum can greatly reduce the overall time [`np.einsum`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.einsum.html) takes by optimizing the expression's contraction order and dispatching many operations to canonical BLAS routines. See the [**documentation**](http://optimized-einsum.readthedocs.io) for more information.
-
-As well as [`opt_einsum.contract`](https://optimized-einsum.readthedocs.io/en/latest/autosummary/opt_einsum.contract.html#opt-einsum-contract) acting as a drop-in replacement for `np.einsum`, the following capabilities are enabled by `opt_einsum`:
-
-* Inspect [detailed information](http://optimized-einsum.readthedocs.io/en/latest/path_finding.html) about the path chosen.
-* Perform contractions with [various backends](http://optimized-einsum.readthedocs.io/en/latest/backends.html), including on the GPU and with libraries such as [TensorFlow](https://www.tensorflow.org) and [PyTorch](https://pytorch.org).
-* Generate [reusable expressions](http://optimized-einsum.readthedocs.io/en/latest/reusing_paths.html), potentially with [constant tensors](http://optimized-einsum.readthedocs.io/en/latest/reusing_paths.html#specifying-constants), that can be compiled for greater performance.
-* Use an arbitrary number of indices to find contractions for [hundreds or even thousands of tensors](http://optimized-einsum.readthedocs.io/en/latest/ex_large_expr_with_greedy.html).
-* Share [intermediate computations](http://optimized-einsum.readthedocs.io/en/latest/sharing_intermediates.html) among multiple contractions.
-* Compute gradients of tensor contractions using [autograd](https://github.com/HIPS/autograd) or [jax](https://github.com/google/jax)
-
-## Quick tutorial
-Einsum is a powerful function for contracting tensors of arbitrary dimension and index.
-However, it is only optimized to contract two terms at a time resulting in non-optimal scaling.
-
-For example, let us examine the following index transformation:
-`M_{pqrs} = C_{pi} C_{qj} I_{ijkl} C_{rk} C_{sl}`
+Optimized einsum can greatly reduce the overall execution time of einsum-like expressions (e.g.,
+[`np.einsum`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.einsum.html),
+[`dask.array.einsum`](https://docs.dask.org/en/latest/array-api.html#dask.array.einsum),
+[`pytorch.einsum`](https://pytorch.org/docs/stable/torch.html#torch.einsum),
+[`tensorflow.einsum`](https://www.tensorflow.org/api_docs/python/tf/einsum),
+)
+by optimizing the expression's contraction order and dispatching many
+operations to canonical BLAS, cuBLAS, or other specialized routines. Optimized
+einsum is agnostic to the backend and can handle NumPy, Dask, PyTorch,
+Tensorflow, CuPy, Sparse, Theano, Jax, and Autograd arrays. See the
```

Comment:
Might be worth saying "as well as potentially many others" or something similar. These are the explicitly tested backends but others should work as well!